### ISSUE TITLE
[CPU Cycle Exact] addition to XML

### ISSUE THREAD

**Issue by nemo93** [midwan/amiberry#959]
TL;DR: for Amiberry to parse `CPU_EXACT=FALSE/TRUE` from XML. It's set only when `CPU=68000` and for a limited # of games.

Provided the new CPU "Cycle Exact" parameter does help with some games (eg. _Ikari Warriors_ has graphics glitch represented by few lines of static at the bottom of the screen), let's add that parameter to XML. This improves emulation accuracy as well. As for the other XML parameters this allows fine-tuning given it could be switched on/off on a per-game basis.

This setting will apply only when CPU model is equal to 68000. It's worth noting that when _turned on_ this parameter will improve accuracy at the cost of _more_ CPU power. On a Pi4 impact is negligible (no noticeable slow down for instance) but it might be a different story on Pi3 and below. This explains why the setting is _turned off_ by default.

As of typing only a very limited number of games does require that setting to be turned on. Therefore impact for users should be minimal.

* XML test file: available [there](https://raw.githubusercontent.com/nemo93/Amiberry-XML-Builder/1b6de1655109bf13142a19a17f0cdc51f32d186d/whdload_db.xml)
* List of games: [here](https://github.com/HoraceAndTheSpider/Amiberry-XML-Builder/blob/master/settings/CPU_CycleExact.txt). Ikari Warriors is the best candidate as the gfx glitch can be seen right from the intro screen.

**Comment by midwan** [comment]
@nemo93 
This is now added in `master` - I tested it and it seems to work as expected, but it would be great if you could do another test from your side also. :)

**Comment by nemo93** [comment]
@midwan thanks. Working too on my end. Yet the glitch described in OP is still there in Ikari. There's also a timing issue: on dying the character will spin on himself but you can see it does so at a high speed. Weird given in the previous build turning `exact cycle` fixed it for good.
